specification/paths/MultiColliShipment.json

```diff
@@ -0,0 +1,52 @@
+{
+  "post": {
+    "tags": [
+      "Multi-colli shipment"
```

Comment:
I would say keep it plural like all other endpoints: `Multi-colli shipments`

Reply:
Makes sense! :ok_hand: